cla_public/apps/base/filters.py

```diff
@@ -14,3 +16,17 @@ def datetime(dt, format='medium', locale='en_GB'):
     elif format == 'short':
         format = "dd/MM/y, h:mma"
     return format_datetime(dt, format, locale=locale)
+
+
+@base.app_template_filter()
+def humanize_url(value, trim_url_limit=None, nofollow=None, target=None):
+    pattern = re.compile('(^https?:\/\/)|(\/$)')
```

Comment:
if you use a regex string `r'(^https?://)|(/$)'`, you don't need to escape things